brainframe/cli/config.py

```diff
@@ -55,13 +52,12 @@ def load(
 
 def load() -> None:
     """Initializes configuration options"""
-    if not _DEFAULTS_FILE_PATH.is_file():
-        print_utils.fail_translate(
-            "general.missing-defaults-file",
-            defaults_file_path=_DEFAULTS_FILE_PATH,
-        )
+    try:
+        defaults_file_path = frozen_utils.defaults_file_path()
+    except frozen_utils.ResourceNotFoundError:
+        print_utils.fail_translate("general.missing-defaults-file")
```

Comment:
I'm a bit confused, what does this code do? If the `defaults_file_path` doesn't exist, I can only assume that line 60 will fail?

Reply:
`print_utils.fail_translate` prints the string corresponding to the translation ID and exits with a non-zero exit code, so line 60 will never be reached.